Fortnox.NET/WebSockets/Models/WebSocketResponse.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+
+namespace Fortnox.NET.WebSockets.Models
```

Comment:
FYI - I Think NewtonSoft handles lower-to-upper case conversion without annotations. I don't mind the specificity of the annotations though!